documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,63 @@
+# Get Bootchart from a Clear Container#
+
+The following steps describe how to get a bootchart from a Clear Cotainers.
+Currently systemd is the default init process in the Clear Containers mini-os
+system. Systemd provides a tool called systemd-bootchart that  that
+collects the CPU load, disk load, memory usage, as well as per-process information
+from the os boot process.
+
+## Configure Clear Containers to get a bootchart##
+
+To get a bootchart from a Clear Container you must vefiry the following 
+steps before:
+
+### Prepare your Clear Container image ###
+
+The file data/cc-bootchart.conf can be used get a bootchart graph.
+
+The file must be installed Clear-Container image using systemd and systemd 
+bootchart.
+
+The file data/cc-bootchart can be installed with the configure option
+--with-cc-image-bootchart-config-dir=DIR  where DIR is a directory
+to any of the following paths:
+
+```
+/etc/systemd/bootchart.conf
+/etc/systemd/bootchart.conf.d/*.conf
+/run/systemd/bootchart.conf.d/*.conf
+/usr/lib/systemd/bootchart.conf.d/*.conf
+```
+
+Note : Any any of the paths above must inside the container image.
+
+### Configure Clear Containers to boot with bootchart ###
+
+Make sure that you kernel use as init proccess as systemd-bootchart
+
+To speficify the init binary modify the file vm.json the file:
+
+Append to the "parameters" json node "init=PATH-TO-SYSTEMD-BOOTCHART"
+( usually  init=/usr/lib/systemd/systemd-bootchart)
+
+
+```
+/etc/cc-oci-runtime/vm.json
+```
+or  the defaults system configuration file:
+```
+/usr/share/defaults/cc-oci-runtime/vm.json
+```
+
+## Extract bootchart from a container ##
+
+After conifgure your Clear Container image and init binary and 
+container will have a bootchart in the path /run/bootchart-\*.svg
+
+Example:
+
+```
+# wait some time before try to read bootchart.
```

Comment:
I'd explain why you need to wait here. It would be great if we could get rid of that `sleep`.